### PR TITLE
Revs is now delayed by 5 minutes

### PR DESCRIPTION
**Pull request by ShiftyRail**
Hopefully the rounds will now last 15 minutes instead of 10
[tweak]

:cl:
- tweak: Roundstart revs is now delayed by 5 minutes

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
GRUDGECODE
But seriously the sudden initiative from revs is what allows rev rounds to end quickly before command and security digs in place and gradually gets rid of revolutionaries in a terribly slow slog until it ends in a theorycrafted incident where revs bomb the whole thing or the real deal events in which the shuttle arrives or they all manage to kill the head revolutionaries as they kill & hold or grab & implant people. Not sure if the shuttle can depart though

**Comment by ShiftyRail** [comment]
In an ideal world it would be changeling'd

**Comment by boy2mantwicethefam** [comment]
In an ideal world SS13 wouldn't be on BYOND

File: code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -348,23 +348,25 @@
...
+			var/mob/M = pick(candidates)
+			assigned += M
+			candidates -= M
+			var/datum/role/revolutionary/leader/lenin = new
+			lenin.AssignToRole(M.mind, 1, 1)
+			R.HandleRecruitedRole(lenin)
+			lenin.Greet(GREET_ROUNDSTART)
+		update_faction_icons()
+		R.OnPostSetup()
+		return 1
```
Comment (MadmanMartian) [inline code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm]: Tab this back once, as execute is expected to return 1, else the ruleset is dumped.

**Review by MadmanMartian — CHANGES_REQUESTED** [review]
do the thing

**Review by MadmanMartian — APPROVED** [review]

**Pull request merged**